vllm/distributed/utils.py

```diff
@@ -150,6 +153,15 @@ def recv_obj(self, src: int) -> Any:
         self.recv_src_counter[src] += 1
         return obj
 
+    def recv(self, tensor: torch.Tensor, src: int):
```

Comment:
if you just recv a tensor, and you already know the shape with a given buffer `tensor`, either `pynccl.send` or `torch.distributed.send` should just work. No need to go through `send_obj`

Reply:
Fixed -- thanks!